mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go

```diff
@@ -0,0 +1,124 @@
+package securitycenterv2_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-google/google"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+	"google.golang.org/api/googleapi"
+)
+
+func TestAccSecurityCenterOrganizationMuteConfig_createUpdateDelete(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":        envvar.GetTestOrgFromEnv(t),
+		"random_suffix": acctest.RandString(t, 10),
+		"location":      "global",
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck: func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: map[string]func() (*schema.Provider, error){
```

Comment:
What are you using as a reference for this line? Please see other tests, which use a predefined factory: https://github.com/hashicorp/terraform-provider-google-beta/blob/bfb8b9067be9965baf389e312815aaa34056fba8/google-beta/services/compute/resource_compute_address_test.go#L35

Reply:
addressed